test/SlowTests/Issues/RavenDB_23448.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Linq;
+using FastTests;
+using Orders;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.ServerWide.Operations;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_23448(ITestOutputHelper output) : RavenTestBase(output)
+{
+    [RavenFact(RavenTestCategory.Vector | RavenTestCategory.Querying)]
+    public void CanQueryTextualValuesInVectorSearchWhenFieldConfigurationIsNotExplicitlySet()
+    {
+        using var store = GetDocumentStore(new Options() { RunInMemory = false, DeleteDatabaseOnDispose = true, Path = NewDataPath() });
+        new Index().Execute(store);
+
+        using (var session = store.OpenSession())
+        {
+            session.Store(new Product { Name = "Product 1" });
+            session.SaveChanges();
+            Indexes.WaitForIndexing(store);
```

Comment:
At this point there is nothing to index, isn't it? You should rather use WaitForNonStaleResults in below queries

Reply:
This applies to other tests as well

Reply:
Sorry, I didn't notice above `new Index().Execute(store)`